docs/cooperation/cooperation-schema.yaml

```diff
@@ -210,10 +154,210 @@ definitions:
                   type: string
                   format: date-time
                   default: null
+                  nullable: true
             completionStatus:
               $ref: '#/definitions/completionStatus'
   completionStatus:
     type: string
     enum:
       - completed
       - active
+  cooperationStatusEnum:
+    type: string
+    enum:
+      - pending
+      - active
+      - declined
+      - closed
+      - request to close
+  resourceType:
+    type: string
+    enum:
+      - lesson
+      - quiz
+      - attachment
+      - question
+  userInCooperations:
+    type: object
+    properties:
+      _id:
+        type: string
+      firstName:
+        type: string
+      lastName:
+        type: string
+      photo:
+        type: string
+      role:
+        $ref: '#/definitions/baseUserRoleEnum'
+  userInCooperationsExtended:
```

Comment:
isn't this user the same as a default user? in this case, you can avoid creating new schema and use the default user one

Reply:
The main user schema and the user schema in cooperation are different, so I created a separate one